docs/DeveloperGuide.adoc

```diff
@@ -122,27 +122,26 @@ image::ModelClassDiagram.png[]
 The `Model`,
 
 * stores a `UserPref` object that represents the user's preferences.
-* stores the Address Book data.
-* exposes an unmodifiable `ObservableList<Person>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
+* stores the `DriverManager`, `TaskManager`, `CustomerManager`
+* exposes unmodifiable `ObservableList` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
 * does not depend on any of the other three components.
 
-[NOTE]
-As a more OOP model, we can store a `Tag` list in `Address Book`, which `Person` can reference. This would allow `Address Book` to only require one `Tag` object per unique `Tag`, instead of each `Person` needing their own `Tag` object. An example of how such a model may look like is given below. +
- +
-image:BetterModelClassDiagram.png[]
 
 [[Design-Storage]]
 === Storage component
 
 .Structure of the Storage Component
 image::StorageClassDiagram.png[]
+.Structure of the CentralManager
+image::CentralManager.png[]
```

Comment:
The CentralManager is by itself on the top right hand corner and s not connected to anything else in the diagram.
![Screen Shot 2019-10-25 at 11 36 06 AM](https://user-images.githubusercontent.com/37840339/67541542-dd456100-f71b-11e9-8bf9-b29f7881b526.png)